driver.py

```diff
@@ -0,0 +1,12 @@
+"""Module for running the Branston Classes"""
+# Author: Daniel Davis
+# Group: CSCK451 Group A
+# Date: 29/09/2023
+
+from cwizard import ClientWizard
+
+
+# Server Wizard Driver Code *******************************************

```

Comment:
This module can be removed by including the following in the client wizard:
[end of cwizard.py file]
"
if __name__ == "__main__":
my_wizard = ClientWizard()
my_wizard.ask_all()
my_wizard.display()
"

Reply:
Driver code will not be a part of the project. I'm just supplying code to run the module so that people can see it functioning.